packages/strapi/lib/middlewares/cron/index.js

```diff
@@ -20,8 +20,16 @@ module.exports = strapi => {
 
     initialize() {
       if (strapi.config.get('server.cron.enabled', false) === true) {
+        const tz = strapi.config.get('server.cron.timeZone', 'US/Eastern');
```

Comment:
We should not be setting a default here, or it should default to the system time (which normally will be UTC)

Reply:
Right, should pass in `null` instead so that cron-parser [handles this as the system timezone](https://github.com/harrisiirak/cron-parser/blob/master/lib/date.js#L197) instead of setting a default timezone.